src/lib.rs

```diff
@@ -133,6 +133,7 @@ pub mod test;
 
 #[cfg(test)]
 mod tests {
+    #[cfg(feature = "rsa_signing")]
```

Comment:
We still need to run the BIGNUM tests because RSA verification still uses BIGNUM (and ECC too, for a short while still.)